src/zeit/content/cp/interfaces.py

```diff
@@ -290,6 +302,22 @@ class IQueryConditions(zeit.content.article.interfaces.IArticle):
         source=zeit.cms.content.interfaces.authorSource,
         required=False)
 
+    # ICommonMetadata has ressort and sub_ressort in separate fields, but we
+    # need them combined. And so that whitespace-separated serializing works,
+    # we wrap it in a tuple to reuse the DAVPropertyConverter for `channels`.
+    ressort = zope.schema.Tuple(value_type=zc.form.field.Combination(
+        (zope.schema.Choice(
+            title=_('Ressort'),
+            source=zeit.cms.content.sources.RessortSource()),
+         zope.schema.Choice(
+             title=_('Sub ressort'),
+             source=QuerySubRessortSource(),
+             required=False)),
+        default=(),
+        required=False))
+    zope.interface.alsoProvides(
+        ressort.value_type, zeit.cms.content.interfaces.IChannelField)
```

Comment:
Also hmm, das ist zwar elegant würde ich sagen, aber nicht auch langsam schon so indirekt, daß wir da vielleicht nochmal drübergehen sollten, also bei Gelegenheit…  oder anders gefragt, versteht das noch jemand in einem Jahr, also außer dir? 😄